test/models/tf_ov_model_zoo/gen_plot.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# ==============================================================================
+#  Copyright 2018-2020 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+import argparse
+import errno
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--csv',
+        type=str,
+        help="Input CSV file for genearting chart.\n",
```

Comment:
Typo: "generating".

Minor, but you can change occurrences of "chart" in help to "plot".